src/feed/filters.py

```diff
@@ -50,6 +53,48 @@ def filter_queryset(self, request, queryset, view):
         # Fall back to default ordering behavior for any explicit ordering parameters
         return super().filter_queryset(request, queryset, view)
     
+    def _apply_include_ended_filter(self, request, queryset, view):
+        """
+        Apply include_ended filtering to exclude OPEN items past their end_date.
+        
+        Args:
```

Comment:
Let's avoid long comments as they make the code pretty hard to read. A lot of the times the auto-gen comments don't add meaningful value. Using type annotations in python is better (see comment in slack)